vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -568,7 +568,11 @@ def allowed_templates(options = {})
       _log.warn "Allowed Templates is returning an empty list"
     else
       _log.warn "Allowed Templates is returning <#{allowed_templates_list.length}> template(s)"
-      allowed_templates_list.each { |vm| _log.info "Allowed Template <#{vm.id}:#{vm.name}>  GUID: <#{vm.guid}>  UID_EMS: <#{vm.uid_ems}>" }
+      if _log.debug?
+        allowed_templates_list.each do |vm|
+          _log.info "Allowed Template <#{vm.id}:#{vm.name}>  GUID: <#{vm.guid}>  UID_EMS: <#{vm.uid_ems}>"
```

Comment:
@matthewd is correct.  The templates are already loaded so there is not much of a performance hit here, we are just trying to avoid flooding the log with these messages.  This change could be reduced to just switching the log level to debug.